### ISSUE TITLE
Question regarding "unknown decode filter"

### ISSUE THREAD

**Issue by Lolf1010** [veraPDF/veraPDF-library#1481]
Hi everyone,

at work we use veraPDF to validate pdf files uploaded by our users of our platform.
We have seen the log-message "Unknown decode filter" a couple of times recently.
The log comes from `org.verapdf.cos.filters.COSFilterRegistry#getDecodeFilter`, see [here](https://github.com/veraPDF/veraPDF-parser/blob/integration/src/main/java/org/verapdf/cos/filters/COSFilterRegistry.java#L95).

**Questions**
Is it possible to add the name of the filter to the log-message so that we can see which unsupported filter was used?

Are there any known filters that are currently not supported by veraPDF?
I have found a couple online that are not in the registry (RunLengthDecode, JBIG2Decode, DCTDecode) but they did not cause the log-message. I probably did not use them correctly or, even more likely, dont understand that part of pdf-files completely ;)

And, since the log-level is SEVERE: should we report unsupported filters to you so they can be added or can they be ignored when validating a pdf-file?

Thanks.

**Comment by MaximPlusov** [comment]
Included into release 1.28

**Comment by mkl-public** [comment]
> veraPDF does not decode images and as such does not support image-specific filters: `RunLengthDecode`, `CCITTFaxDecode`, `JBIG2Decode`, `DCTDecode`, `JPXDecode`.

`RunLengthDecode` is not image-specific.

**Comment by bdoubrov** [comment]
> `RunLengthDecode` is not image-specific.

Good catch! We'll add RunLength decoder just in case it is used for data relevant for validation.

**Comment by MaximPlusov** [comment]
Fixed in the latest dev build.

- the RunLengthDecode filter is supported
- the Crypt filter without decode parameters is supported

**Comment by MaximPlusov** [comment]
Included into release 1.30.1
